src/System.Transactions.Local/src/System.Transactions.Local.csproj

```diff
@@ -8,7 +8,7 @@
     There are many instances of comparing an instance of a struct with null. Until these are all fixed,
     disable the diagnostic.
     -->
-    <Features>$(Features.Replace('strict', '')</Features>
```

Comment:
Delete this instead (including the comment). This is enabled globally for corefx.

Reply:
Done